qiskit/transpiler/transpile_circuit.py

```diff
@@ -14,69 +14,22 @@
 
 """Circuit transpile function"""
 
-from qiskit.transpiler.preset_passmanagers import (level_0_pass_manager,
-                                                   level_1_pass_manager,
-                                                   level_2_pass_manager,
-                                                   level_3_pass_manager)
-from qiskit.transpiler.passes.basis.ms_basis_decomposer import MSBasisDecomposer
-from qiskit.transpiler.exceptions import TranspilerError
 
-
-def transpile_circuit(circuit, transpile_config):
+def transpile_circuit(circuit,
+                      pass_manager,
+                      output_name: None,
+                      callback: None):
     """Select a PassManager and run a single circuit through it.
```

Comment:
This no longer "selects" a PassManager. The PassManager is given. 

Again, please thoroughly read all the docstrings for the affected modules and functions because a lot of old behavior is changing here.

Reply:
I actually think this whole file can be removed. The point of this `transpile_circuit()` function was to provide a thin layer of abstraction over `PassManager.run()`, and handle the selection of pass manager. Now the pass manager is given, and this function does nothing but call `PassManager.run()`. 

You just have to make parallel_map call `PassManager.run(circuit)` rather than `transpile_circuit(circuit)`

Reply:
`transpile_circuit` is part of the API, I can remove in a coming PR. The docstring was updated here: 346b8ea21

Reply:
`transpile_circuit()` has already changed in this PR. It used to take a `TranspileConfig` and no longer does.

Reply:
Done in 8d935be56